lib/kafka/gzip_codec.rb

```diff
@@ -0,0 +1,28 @@
+module Kafka
+  class GzipCodec
```

Comment:
There's no reason for subclassing anything here, there's no need for shared behavior.